das/daser.go

```diff
@@ -140,37 +136,17 @@ func (d *DASer) Stop(ctx context.Context) error {
 	if err = d.sampler.wait(ctx); err != nil {
 		return fmt.Errorf("DASer force quit: %w", err)
 	}
-
-	// save updated checkpoint after sampler is shut down
-	if err = d.cstore.store(ctx, newCheckpoint(d.sampler.state.unsafeStats())); err != nil {
-		log.Errorw("storing checkpoint to disk", "Err", err)
-	}
-
```

Comment:
Why is this removed? The double logging issue should not be a reason for us to resample heights that were finished during `wait` stage.

Reply:
returned it. I've suspected it could have had concurrency issues, but it is fine actualy.